R/stratify_data.R

```diff
@@ -8,11 +11,14 @@ stratify_data <- function(
     noise_strata_varnames = NULL)
 {
 
+
   if(is.null(strata_varnames) || all(strata_varnames == ""))
   {
     all_data =
       list(
-        pop_data = data %>% select("value", "age", "antigen_iso"),
+        pop_data = est_data %>% select("value",
```

Comment:
what happens when `standardize = FALSE`? 

Please add a unit test for this function.

We should be pulling these strings out of the attributes of `est_data`

Reply:
When `standardize = FALSE` the user-provided column names remain. There's no need to add additional code to cater for the condition when `standardize = FALSE`. I will add unit test to check the results when `FALSE`. On the pulling string from `est_data`, I will make the change.